### ISSUE TITLE
README.MD is extremely out of date

### ISSUE THREAD

**Issue by Saracevas** [ortuman/SwiftForms#142]
Examples given in the README.MD no longer work and require editing afterwards. As someone who's new to Swift, it makes it difficult to adopt and use the framework. Although I'm grateful for all hard work you put into this 👍

**Comment by AlexanderBollbach** [comment]
I agree.  I'm inheriting a project which used the old style and am clueless about how to proceed.
